packages/core/src/interpreter.ts

```diff
@@ -1010,8 +1036,12 @@ export class Interpreter<
           try {
             // Send "error.platform.id" to this (parent).
             this.send(toSCXMLEvent(errorEvent as any, { origin: id }));
-          } catch (error) {
-            reportUnhandledExceptionOnInvocation(errorData, error, id);
+          } catch (errorFromSend) {
+            if (this.machine.strict) {
+              this.handleError(errorFromSend);
+            }
+
+            reportUnhandledExceptionOnInvocation(errorData, errorFromSend, id);
```

Comment:
I'm not sure what exact role does `reportUnhandledExceptionOnInvocation` serve now and what's the role of the `defaultErrorHandler`, they seem to be overlapping and most likely they can both be executed for the same error - leading to excessive logs in the console.

Reply:
`defaultErrorHandler` has been removed.